src/main/target/SPRACINGF3/target.h

```diff
@@ -94,7 +94,7 @@
 #define ACC_MPU6050_ALIGN       CW270_DEG
 #endif
 
-#if defined(FLIP32F3OSD)
+#if defined(SPRACINGF3) || defined(FLIP32F3OSD)
```

Comment:
I concur with @copter58, this will not work since `SPRACINGF3` is defined for all variants. Best way is probably to turn the `#if` statement around, and put SPRACINGF3 / FLIP32F3OSD into the `#else` clause.